spec/support/client.rb

```diff
@@ -0,0 +1,3 @@
+def client
+  Myfinance.client('b552dd5a8598ca089b91c5e355a83b86e4696aefac9eaa05')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.